scripts/tune_pid.py

```diff
@@ -28,26 +28,27 @@
 
 # Author: Stuart Glaser
 
-import sys, time
-
-import rospy
-import control_toolbox.srv
-
-def print_usage(exit_code = 0):
-    print "Usage:  tune_pid.py <service> [<p> <i> <d> <i_clamp>]"
-    sys.exit(exit_code)
-
-def main():
-    if not (len(sys.argv) in [2, 6]):
-        print_usage()
-
-    if len(sys.argv) == 2:
-        print "Not implemented yet"
-    elif len(sys.argv) == 6:
-        service_name = sys.argv[1]
-        if not service_name.endswith('/set_gains'):
-            service_name = service_name + '/set_gains'
-        set_gains = rospy.ServiceProxy(service_name, control_toolbox.srv.SetPidGains)
-        set_gains(*map(float, sys.argv[2:6]))
-
-if __name__ == '__main__': main()
+# @TODO adapt this to ROS2
```

Comment:
I'd either entirely rewrite this with proper argparse, or remove it altogether and delay rewriting until we need it

Reply:
Should I just delete the file and all trace to it entirely then?

Reply:
let the past die, kill it, if you have to:+1:

Reply:
Killed.